msal/tests/Test.MSAL.NET.Unit/ModuleInitializerTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Core;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Test.MSAL.NET.Unit
+{
+    [TestClass]
+    public class ModuleInitializerTest
+    {
+        [TestMethod]
+        public void InitializesExceptionsAndLogs()
+        {
+            // Act
+            ModuleInitializer.EnsureModuleInitialized();
+
+            // Assert
+            MsalExceptionFactory factory = CoreExceptionService.Instance as MsalExceptionFactory;
```

Comment:
we are back to the question on naming from another PR... perhaps both should be called service?

Reply:
This will have been fixed in the next PR related to ADAL Exception Service